scenarios/kubernetes_aws.py

```diff
@@ -208,6 +204,21 @@ def main(args):
         '--up', default='true', help='If we need to set --up in e2e.go')
     ARGS = PARSER.parse_args()
 
+    # If aws keys are missing, try to fetch from HOME dir
+    if not (ARGS.aws_ssh or ARGS.aws_pub or ARGS.aws_cred):
+        HOME = os.environ.get('HOME')
+        if not HOME:
+            raise ValueError('HOME dir not set!')
+        if not ARGS.aws_ssh:
+            ARGS.aws_ssh = '%s/.ssh/kube_aws_rsa' % HOME
+            print >>sys.stderr, 'AWS ssh key not found, try to fetch from %s' % ARGS.aws_ssh
```

Comment:
I prefer the bikeshed reads Try actually, since progressive tense is weak :grin: